### PR TITLE
6 implement client tests

### LINKED ISSUE #6: Build out tests
Generate verbose tests to validate client functionality

### PR DESCRIPTION
**Pull request by vardyb**
https://waffle.io/AgileVentures/asyncvoter-slack-command/cards/5a5c8548fc5bb8006be8ecf2

Fixes #6

### CODE REVIEW

**Comment by tansaku** [comment]
looking good @vardyb :-)

File: spec/story/story.spec.js [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+        this.timeout(2500);
+        client.setBaseUrl('http://api-test.asyncvoter.agileventures.org');
+        client.createStory(newStory, function (err, data, response) {
+            storyId = data._id;
+            done();
+        });
+    });
+
+    it('should return a code 200', function (done) {
+        client.getStory(storyId, function (err, data, response) {
```
Comment (tansaku) [inline spec/story/story.spec.js]: in this test and below we could refactor to get all the data up front and then test with one liners, like we do at the top of the file

File: spec/vote/vote.spec.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        this.timeout(2500);
+        client.setBaseUrl('http://api-test.asyncvoter.agileventures.org');
+        client.createVote(storyId, newVote, function (err, data, response) {
+            newVoteId = data._id;
+            done();
+        });
+    });
+
+    it('should return a code 200', function (done) {
+        client.getVotesForStory(storyId, function (err, data, response) {
```
Comment (tansaku) [inline spec/vote/vote.spec.js]: in this test and below we could refactor to get all the data up front and then test with one liners, like we do at the top of the file

**Comment by vardyb** [comment]
@tansaku thanks for the feedback I'll hopefully have time next week to look at this and open a refactoring ticket.

**Comment by vardyb** [comment]
@tansaku - Thanks again for the feedback, I've refactored the two spec files in questions can you please review again?

**Comment by tansaku** [comment]
looks great @vardyb - many thanks for this :-)

**Pull request merged**